scenarios/arcaflow/sysbench/input.yaml

```diff
@@ -0,0 +1,5 @@
+kubeconfig: {}
```

Comment:
Can we read the kubeconfig from the location specified in the Kraken config vs having to define in this config? Thoughts?